02-esc/challenges.js

```diff
@@ -184,18 +185,21 @@ function showAllChallenges(resultFromAPI) {
 		btnBook.classList.add("btnBook", "button", "primary");
 
 		btnBook.setAttribute("data-id", id);
+		btnBook.setAttribute("data-participants", minParticipants);
+		btnBook.setAttribute("data-participants", maxParticipants);
```

Comment:
These two lines of code first set an attribute, and then reset it with another value. This means that line is 188 is without effect (or rather, it's effect is made obsolete immediately by the next line).

Reply:
Oooops. I can't find it in chellnges.js at line 188. But point taken though.